security/subsystem/src/main/java/org/jboss/as/security/deployment/SecurityAttachments.java

```diff
@@ -31,11 +31,16 @@
 
     /**
      * Attachment key that if present signifies that the security subsystem is installed.
-     *
+     * <p>
      * If this is not present either the subsystem has been removed from the config or excluded
      * via jboss-deployment-structure.xml. This allows deployments to disable security, and
      * avoid the overhead of running unneeded security code.
+     *
+     * @deprecated Check the presence of {@link org.jboss.as.security.SecuritySubsystemRootResourceDefinition#SECURITY_SUBSYSTEM}
```

Comment:
Please use the string form of the capability name instead of a link to a constant. We don't want to encourage people to import constants from other modules.

Reply:
Done in https://github.com/wildfly/wildfly/pull/11763